src/cli/Cli.spec.ts

```diff
@@ -2176,6 +2183,66 @@ describe('Cli', () => {
       });
   });
 
+  it('shows help for command in browser when --help option specified', (done) => {
```

Comment:
Let's clarify in the name of the test that we're assuming configuration to show help in the browser